weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/callsTableColumns.tsx

```diff
@@ -637,3 +510,146 @@ const isExpandedRefWithValueAsTableRef = (
   }
   return isTableRef(ref[EXPANDED_REF_VAL_KEY]);
 };
+
+export const buildDynamicColumns = <T extends GridValidRowModel>(
```

Comment:
There is no code change here, just extracting this function to share with ObjectVersions page.